README.md

```diff
@@ -1,2 +1,52 @@
 # 2019-winter-Haskell-school
+
 2019 winter Haskell school in Sydney Australia 
+
+
+
+## Class timetables and planning
+
+This winter session is initially designed to have 10 classes(might extended).
+
+Each class has average length of 2 hours. Each week, we will pick a time and place and announce it.
+
+
+
+## Materials that we used for this class
+
+
+
+Currently we are using [Yorgey's cis194 course](https://github.com/bitemyapp/learnhaskell#yorgeys-cis194-course). 
```

Comment:
There are a few different versions. We should link which version we are using

Reply:
done